tools/fuchsia/fuchsia_archive.gni

```diff
@@ -149,6 +150,14 @@ template("_fuchsia_archive") {
       "--manifest-json-file",
       rebase_path(manifest_json_file, root_build_dir),
     ]
+
+    if (fuchsia_target_api_level != -1) {
```

Comment:
sorry, I'm still struggling with GN :( where does this variable come from? Do all gn_args[] get put into the namespace as variables automatically?

Reply:
It comes from the import above where we import //build/fuchsia/config.gni. This is a gn thing, when you put a variable in a declare_args it makes it available as a global after an import.